test/core/Do.Test.Architecture/Architecture/Application/RunningAnApplication.cs

```diff
@@ -41,8 +41,8 @@ public void Each_phase_is_applied_separately_to_all_layers()
     {
         var phase1 = MockMe.APhase();
         var phase2 = MockMe.APhase();
-        var layer1 = MockMe.ALayer(phase: phase1);
-        var layer2 = MockMe.ALayer(phase: phase2);
+        var layer1 = MockMe.ALayer(id: "Do.Test.Architecture.Application.LayerA", phase: phase1);
+        var layer2 = MockMe.ALayer(id: "Do.Test.Architecture.Application.LayerB", phase: phase2);
```

Comment:
use guid when mocking a layer to avoid setting a unique id where it is irrelevant